php/project/templates/vhost.conf

```
@@ -0,0 +1,41 @@
+upstream {{ serverName }} {
+{% for item in upstreams %}
```

si tu veux éviter d'avoir des lignes vides à cause des mot-clefs Twig, tu prux ajouter des `-` pour faire un équivalent de `ltrim` ou `rtrim` (comme l'exemple ci-dessous) :
```suggestion
{% for item in upstreams -%}
```